src/ploomber_core/telemetry/telemetry.py

```diff
@@ -426,6 +427,11 @@ def validate_entries(event_id, uid, action, client_time, total_runtime):
     return event_id, uid, action, client_time, elapsed_time
 
 
+def is_first_arg_self(func):
+    params = list(inspect.signature(func).parameters)
+    return len(params) > 0 and "self" in params and params[0] == "self"
```

Comment:
do we need the `"self" in params`?, `params[0] == "self"` is already checking that

Reply:
You're right